contrib/pkg/broker/controller/controller.go

```diff
@@ -25,7 +25,7 @@ type Controller interface {
 	Catalog() (*brokerapi.Catalog, error)
 
 	GetServiceInstance(id string) (string, error)
-	CreateServiceInstance(id string, req *brokerapi.ServiceInstanceRequest) (*brokerapi.CreateServiceInstanceResponse, error)
+	CreateServiceInstance(id string, req *brokerapi.CreateServiceInstanceRequest) (*brokerapi.CreateServiceInstanceResponse, error)
```

Comment:
I know it a minor thing, but I do really like having the word "Create" in there to help identify the exact message exchange we're dealing with.